_dev/src/components/product-feed/settings/delivery-time-and-rates/estimate-method/custom-carrier-form.vue

```diff
@@ -3,7 +3,7 @@
     <p class="h3 mr-2 mb-2 font-weight-600 d-inline-block">
       {{ $t('productFeedSettings.steps.deliveryTimesAndRates') }}
     </p>
-    <b-card class="mb-2">
+    <b-card class="mb-2 p3">
```

Comment:
As I was saying ...

```suggestion
    <b-card class="mb-2 p-3">
```

Reply:
Yes my bad, I forgot the - for use bootstrap classes , i didn't knew of p-3 is equal to 15px